README.md

```diff
@@ -27,6 +27,23 @@ centrally managing routing policy and fronting service discovery.
 * [linkerd](linkerd/README.md)
 * [namerd](namerd/README.md)
 
+## Quick start ##
+
+### Boot linkerd ###
+
+```
+$ ./sbt linkerd-examples/http:run
+$ open http://localhost:9990
+```
+
+### Boot namerd ###
+
+```
+$ ./sbt namerd-examples/basic:run
+$ open http://localhost:9991
+$ curl :4180/api/1/dtabs
+```
```

Comment:
i think it's fine to be here, i just think it would be better with a little explanation if it's displayed so prominently in the docs.  (I worry that the README is getting to be a bit overwhelming)